src/markdown-pages/docs/ecosystem/galasa-cli.md

```diff
@@ -0,0 +1,153 @@
+
+All the tests in the `test.yaml` portfolio will run on the z/OS LPAR `MV2C` in the `PLEX2` cluster when the `galasactl runs submit --portfolio test.yaml` command is run, regardless of the LPAR and cluster that is specified in the CPS properties file.
+
+```
+galasactl runs prepare
+                        --portfolio test.yaml
+                        --stream BestSoFar
+                        --package dev.galasa.simbank.tests
+                        --override zos.default.lpar=MV2C
+                        --override zos.default.cluster=PLEX2
+```
+
+### Building a portfolio over multiple selections and overrides
+
+In the following example, the first command creates a portfolio called `test.yaml` that contains tests from the `BestSoFar` test stream where the test package name is `dev.galasa.simbank.tests`. The second command adds more tests to the `test.yaml` portfolio from the `BestSoFar` test stream where the test package name is `dev.galasa.simbank.tests.two`. 
+
+All the tests in the `test.yaml` portfolio from `dev.galasa.simbank.tests` will run on the z/OS LPAR `MV2C` in the `PLEX2` cluster. The tests from `dev.galasa.simbank.tests.two` will run on the z/OS LPAR `MV2D` in the `PLEX2` cluster when the `galasactl runs submit --portfolio test.yaml` command is run.
+
+```
+galasactl runs prepare 
+                        --portfolio test.yaml
+                        --stream BestSoFar
+                        --package dev.galasa.simbank.tests
+                        --override zos.default.lpar=MV2C
+                        --override zos.default.cluster=PLEX2
+galasactl runs prepare
+                        --portfolio test.yaml
+                        --append
+                        --stream BestSoFar
+                        --package dev.galasa.simbank.tests.two
+                        --override zos.default.lpar=MV2D
+                        --override zos.default.cluster=PLEX2
+```
+
+## Working with the `runs submit` command
+
+The following section provides a subset of examples of how you can use the `runs submit` command to complete various tasks, for example, getting help, submitting tests, and setting overrides.
+
+### Getting help
+
+Use the following commands to get more information about the command and command options.
+
+```
+galasactl --help
+galasactl runs --help
+galasactl runs runs --help
+```
+
+### Submitting tests from a portfolio
+
+The example command submits tests from the `test.yaml` portfolio, and specifies the following settings: 
+```
+galasactl runs submit
+                        --portfolio test.yaml
+                        --poll 5
+                        --progress 1
+                        --throttle 5
+```
+where: 
+- `poll` specifies the frequency in seconds that the CLI polls the Ecosystem for test run status. The default value is `30` seconds. 
```

Comment:
do we want to highlight the defaults,  these may change.    would it be better to say, see galasactl runs submit --help for the defaults?

Reply:
Have removed default value sentences and added wording in the getting help sections to include that you can use that to see the defaults.